### PR TITLE
[charts/csm-installer] CSM common installer release

### PR DESCRIPTION
**Pull request by tdawe**
#### Is this a new chart?

Yes

#### What this PR does / why we need it:
This is the 1.0.0 version of the CSM common installer helm chart

#### Which issue(s) is this PR associated with:

- N/A

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [] Variables are documented in the chart README.md
- [x] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

File: charts/csm-installer/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Dell Community Helm Chart for Dell EMC Container Storage Modules (CSM)
+
+Dell EMC Container Storage Modules (CSM) can be deployed using Helm.
+
+For complete deployment instructions, refer to the [Dell EMC Container Storage Modules (CSM) Github repository](https://github.com/dell/csm).
+
+## Helm Chart Versioning
+
+For an explanation and examples on versioning/releasing the Karavi Observability Helm chart, please see the [contributing guide](../../docs/CONTRIBUTING.md#helm-chart-release-strategy).
```
Comment (gallacher) [inline charts/csm-installer/README.md:19]: References "Karavi Observability".  Is this link still even valid? If so is the content valid?

File: charts/csm-installer/values.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (gallacher) [inline charts/csm-installer/values.yaml:1]: Do we have a table in documentation that outlines the helm values?

File: charts/csm-installer/README.md (hunk 1 above)
Comment (tdawe) [reply charts/csm-installer/README.md]: Removed reference to Karavi Observability. Yes, the link and content are still valid detailing how to update versions of a helm chart: https://github.com/dell/helm-charts/blob/main/docs/CONTRIBUTING.md#helm-chart-release-strategy

File: charts/csm-installer/values.yaml (hunk 2 above)
Comment (tdawe) [reply charts/csm-installer/values.yaml:1]: We detail how to update some of the required values (jwtKey, cipherKey, admin credentials) in our common doc portal, but we don't have a full listing. I'll make a note to add a full table with all of the values on that common doc site.

**Review by gallacher — DISMISSED** [review]

File: charts/csm-installer/templates/csm-installer.yaml [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                mode: 0444
+              - key: adminPassword
+                path: admin/ADMIN_PASSWORD
+                mode: 0444
+              - key: dbUserName
+                path: admin/DB_USERNAME
+                mode: 0444
+              - key: dbPassword
+                path: admin/DB_PASSWORD
+                mode: 0444
```
Comment (medegw01) [inline charts/csm-installer/templates/csm-installer.yaml:170]: need newline

**Review by medegw01 — APPROVED** [review]

File: charts/csm-installer/templates/csm-installer.yaml (hunk 3 above)
Comment (tdawe) [reply charts/csm-installer/templates/csm-installer.yaml]: Fixed

**Review by gallacher — APPROVED** [review]

**Pull request merged**